css/styles.css

```diff
@@ -114,6 +114,25 @@ p {
   min-height: 800px;
 }
 
+.background-image {
+  background-position: center;
+  background-size: cover;
+}
+
+.floating-text {
+  background-color: rgba(245, 251, 219, 0.9);
+  padding-block-start: 0.5em;
+  padding-inline: 3em;
+  margin-block: clamp(1.5em, calc(100vw / 20), 5em);
+}
+
+.floating-text .btn {
+  background-color: var(--bs-body-color);
+  color: rgba(245, 251, 219, 1);
+  font-size: 1.2em;
+  margin-inline: 0;
+}
```

Comment:
@amb26 related to your comment about the contrast, it seems these styles or at least some of them aren't being applied for you. Not sure why though.